src/Symfony/Component/Form/Resources/translations/validators.ca.xlf

```diff
@@ -14,6 +14,126 @@
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Per favor introdueix una data vàlida.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Per favor selecciona un arxiu vàlid.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>El camp ocult és invàlid.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Per favor introdueix un enter.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Per favor selecciona un idioma vàlid.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Per favor seleccioneu una configuració regional vàlida</target>
+            </trans-unit>
+            <trans-unit id="114">
+                <source>Please enter a valid money amount.</source>
+                <target>Per favor introdueix una quantitat de diners vàlids.</target>
+            </trans-unit>
+            <trans-unit id="115">
+                <source>Please enter a number.</source>
+                <target>Per favor introdueix un número.</target>
+            </trans-unit>
+            <trans-unit id="116">
+                <source>The password is invalid.</source>
+                <target>La contrasenya es invàlida.</target>
+            </trans-unit>
+            <trans-unit id="117">
+                <source>Please enter a percentage value.</source>
+                <target>Per favor introdueix un valor percentual.</target>
+            </trans-unit>
+            <trans-unit id="118">
+                <source>The values do not match.</source>
+                <target>Els valors no coincideixen.</target>
+            </trans-unit>
+            <trans-unit id="119">
+                <source>Please enter a valid time.</source>
+                <target>Per favor introdueix un temps vàlid.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please select a valid timezone.</source>
+                <target>Per favor selecciona una zona horària vàlida.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please enter a valid URL.</source>
+                <target>Per favor introdueix una URL vàlida.</target>
+            </trans-unit>
+            <trans-unit id="122">
+                <source>Please enter a valid search term.</source>
+                <target>Per favor introdueix un concepte de cerca vàlid.</target>
```

Comment:
This (google) translates to "search concept" is that correct? Is "Introduïu un terme de cerca vàlid." better?

(I dont speak Catalan, so feel free to say no)

Reply:
For me, both options are correct but I choose "concepte" over "terme".

And with "introduïu" you lose the "please" word.

Per favor = Please
Introdueix / Introduïu -> the second is more polite but the same meaning.
Concepte / terme -> Same meaning again but I think the first is better


Roger

Reply:
Okey. Thank you for confirming.